chronos/transports/stream.nim

```diff
@@ -2130,6 +2130,36 @@ proc createStreamServer*(host: TransportAddress,
   createStreamServer(host, StreamCallback2(nil), flags, sock, backlog, bufferSize,
                      child, init, cast[pointer](udata), dualstack)
 
+proc createStreamServer*(flags: set[ServerFlags] = {},
```

Comment:
we need to be able to specify a port here because their typical use in servers is to listen on "any address, an ip version, port XXX".

Reply:
also, without the port, this looks like a constructor for unix domain sockets

Reply:
Fixed in https://github.com/status-im/nim-chronos/pull/512/commits/9101e994591d096bc9df95b0fb2ad7d4447b29a2